src/mobile/src/ui/components/Migration.js

```diff
@@ -107,6 +140,25 @@ class Migration extends Component {
         });
     }
 
+    /**
+     * Changes to a random node
+     *
+     * @method changeNode
+     */
+    changeNode() {
+        this.props.setFullNode(...sample(this.props.nodes, 1));
```

Comment:
[sample](https://lodash.com/docs/4.17.11#sample) takes a single parameter. Perhaps you meant to use [sampleSize](https://lodash.com/docs/4.17.11#sampleSize) here?